api/cron.json

```
@@ -0,0 +1,7 @@
+{
+  "jobs": [
+    {
+      "command": "20 5 * * * npm run cache:refresh"
+    }
```

Je n'arrive pas à trouver a quoi sert le `20` ?

---

Les minutes il me semble, d'ailleurs le job va s'effectuer a 6h20 d'après la doc il faut mettre une heure de moins que l'heure a laquelle on veut qu'il s'exécute, pour 5h20 ca a l'air d'etre plus cela
```suggestion
      "command": "20 4 * * * npm run cache:refresh"
    }
```

---

Actuellement le job est lancé a 6h13 sur la prod. Pas de raisons de changer l'heure a priori.

---

Ah ok my bad alors je pensais que l'on voulait que ca ce produise a 5h20 heure francaise